sklearn/covariance/_shrunk_covariance.py

```diff
@@ -535,17 +535,19 @@ def oas(X, *, assume_centered=False):
     else:
         n_samples, n_features = X.shape
 
-    emp_cov = empirical_covariance(X, assume_centered=assume_centered)
-    mu = np.trace(emp_cov) / n_features
-
+    # Resolves differences with OAS when ussing uncentered data
+    if not assume_centered:
+        X = X - X.mean(0)
+    emp_cov = empirical_covariance(X, assume_centered=True)
```

Comment:
This was done because `OAS(assume_centered=False).fit(X)` produced different results than `oas(X, assume_centered=False)` due to the manual centering in the estimator (required to get the `location_` attribute).

Reply:
I revert this change since we refactor the function. Now it calls the the estimator directly so they should be sync.